### PR TITLE
Align dbot output context path and IN124 problematic packs

### PR DESCRIPTION
**Pull request by YuvHayun**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-10734](https://jira-dc.paloaltonetworks.com/browse/CIAC-10734)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/IsItPhishing/Integrations/IsItPhishing/IsItPhishing.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ display: IsItPhishing
 category: Data Enrichment & Threat Intelligence
 description: Collaborative web service that provides validation on whether a URL is a phishing page or not by analyzing the content of the webpage.
 configuration:
-- display: Server URL (e.g. https://192.168.0.1)
+- display: Server URL (e.g. https://192.168.0.1)The actual score.
```
Comment (JudahSchwartz) [inline Packs/IsItPhishing/Integrations/IsItPhishing/IsItPhishing.yml:9]: ?

File: Packs/illuminate/Integrations/Analyst1/README.md [hunk 2]
```diff
@@ -1168,7 +1250,7 @@ Gets the full JSON for an Analyst1 Indicator given the internal Analyst1 Indicat
             ],
             "malwares": [],
             "originatingIps": null,
-            "path": null,
+            "**Path**": null,
```
Comment (JudahSchwartz) [inline Packs/illuminate/Integrations/Analyst1/README.md:1253]: ?

**Review by JudahSchwartz — APPROVED** [review]
note my comments

**Comment by YuvHayun** [comment]
Need force due to multiple playbook faliures (known issues) @dantavori

**Pull request merged**